frontend/src/metabase/visualizations/lib/settings.js

```diff
@@ -46,17 +50,12 @@ export function getComputedSettings(
       extra,
     );
   }
-  // In modular embedding (react sdk and embed-js) we disable internal click behaviors
-  // but we want to keep external links (EMB-878) and clicking on the cell should fallback to
-  // drills if available (EMB-879)
+
   if (isEmbeddingSdk()) {
-    if (
-      computedSettings.click_behavior &&
-      computedSettings.click_behavior.type === "link" &&
-      computedSettings.click_behavior.linkType !== "url"
-    ) {
-      computedSettings.click_behavior = undefined;
-    }
+    return _.compose(
+      removeInternalClickBehaviors,
```

Comment:
I extracted the old logic into `removeInternalClickBehaviors` and I added a new one for the mapping